### ISSUE TITLE
Teleporting Stunned Mobs

### ISSUE THREAD

**Issue by MissSnowy** [Helios-Empire/Helios-Tracker#112]
So far I have only experienced this with various mobs in Ghostlands that flee using Hammer of Justice.

Just as a mob is starting to flee, hitting them with Hammer of Justice on occasion will make them teleport a good distance away, usually near another mob.

![wowscrnshot_101415_185815](https://cloud.githubusercontent.com/assets/15068526/10501425/7bc84428-72af-11e5-8c91-702a930cec58.jpg)

**Comment by dikkedeur** [comment]
thanks for the report, i thought this one was fixed some time ago, but it seems not 100%.
so i will forward this to the oc guys.

**Comment by dikkedeur** [comment]
is this still happening??

**Comment by dikkedeur** [comment]
i havent seen this happening for some time, can you please get back if this is still the case?
if not then i can close or you can close the issue

**Comment by MissSnowy** [comment]
It still happens from time to time.

**Comment by MissSnowy** [comment]
Had an interesting thing happen today in Silverpine Forest while I was killing some Dalaran Apprentices in Silverpine Forest.

This one was stunned when I hit it with a fireball.. it teleported to another one near by and teleported back to roughly where it was when it died.

**Comment by dikkedeur** [comment]
yeah it is during flee phase that this happens, thats for sure, he will then walk/flee call for assistance to the other npc in his near cell and ask for it, since he is stunned he will do that as teleport,
in fact he should not be teleporting then at all :+1:
